### ISSUE TITLE
End of tabular in a subfile not correctly defined.

### ISSUE THREAD

**Issue by sebistoecklin** [gsalzer/subfiles#17]
Hey there,
related to the issue _Cannot compile tabular #16_, there still seems to be some sort of interference between `tabular` and `subfiles v2.1`:
The end of a tabular environment is not correctly defined.

For example, a left-hand vertical line in a table exceeds the table by one additional line: 

![Error description](https://www.dropbox.com/s/66j8exqfggk2sv0/Screenshot%202020-11-18%20230555.png?raw=1)

The shown table distortion is obtained from the combination of the subfile `chapter1.tex`

```latex
\documentclass[thesis.tex]{subfiles}

\begin{document}

\section{Introduction}

\begin{tabular}{|c c c|}  
    \hline
    a & b & c \\
    \hline
    a & b & c \\
    \hline
\end{tabular}

\end{document}
```
and the main file `thesis.tex`:

```latex
\documentclass{article}
\usepackage{subfiles}

\begin{document}

\subfile{chapter1.tex}

\end{document}
```

When coloring the table, the first cell of this additional line is also colored.

**Comment by sebistoecklin** [comment]
My bad, it is perfectly fixed when manually installing `v2.2` to `C:\Users\[Username]\AppData\Roaming\MiKTeX\tex\latex\subfiles`.

**Comment by gsalzer** [comment]
Great that the problem is solved.
Thanks a lot for reporting the issue!
Even though it was already solved this time, the next time you may be the first and trigger the bug fix.
